src/muse/readers/csv.py

```diff
@@ -181,6 +181,9 @@ def read_io_technodata(filename: Union[str, Path]) -> xr.Dataset:
     from muse.readers import camel_to_snake
 
     csv = pd.read_csv(filename, float_precision="high", low_memory=False)
+    if "Level" not in csv.columns:
+        # Applies to outputs files, where level is not specified (must be "fixed")
+        csv["Level"] = "fixed"
```

Comment:
In principle, this function is applied to both input and output, so how do you know this particular bit of code will apply only to the output file?

Reply:
If the user follows on from one of the example models, then the "Level" column will only be in the inputs file and not the outputs file.

If the outputs file _does_ have this column, then this won't run, but as long as all the values are "fixed" then it doesn't make a difference (and any "flexible" values will get flagged later on anyway).

If the inputs file _doesn't_ have this column, I think it's also fine as "fixed" is reasonable as a default value.

I should probably clarify this in the comment

Reply:
Yeah, I think it is the last point the one that was unclear to me. If that can be explained, it would be great. In general, I think we need to make explicit what any default values in the simulation are.